source/languageHandler.py

```diff
@@ -44,10 +44,23 @@
 LCID_NONE = 0 # 0 used instead of None for backwards compatibility.
 
 LANGS_WITHOUT_TRANSLATIONS: FrozenSet[str] = frozenset(("en",))
-installedTranslation: Optional[weakref.ReferenceType] = None

```

Comment:
Please restore this line - this was probably introduced when merging master.

Reply:
my mistake, fixed